src/ucs/vfs/base/vfs_obj.h

```diff
@@ -53,6 +94,65 @@ void ucs_vfs_obj_add_ro_file(void *obj, ucs_vfs_file_show_cb_t text_cb,
  */
 void ucs_vfs_obj_remove(void *obj);
 
+
+/**
+ * Invalidate VFS node and set method to update the node.
+ * 
+ * @param [in] obj        Pointer to the object to be invalidate.
+ * @param [in] refresh_cb Method to update the node associated with the object.
+ */
+void ucs_vfs_obj_set_dirty(void *obj, ucs_vfs_refresh_cb_t refresh_cb);
```

Comment:
how will it be used?

Reply:
The method allows to set callback to refresh node, e.g. add children. Thus it is possible to create VFS nodes only when they are requested.
Please find usage of the method in gtest `set_dirty_and_refresh`:
1. VFS node is added by `ucs_vfs_obj_add_dir`
2. The first call of `ucs_vfs_path_get_info` shows that appropriate directory is empty
3. Adding refresh callback by `ucs_vfs_obj_set_dirty`
4. The second call of `ucs_vfs_path_get_info` initiates refresh callback which adds child node
5. The second call of `ucs_vfs_path_get_info` shows that appropriate directory has a child